essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md

```diff
@@ -227,48 +310,57 @@ Let's dive in!</p>
+- [Flutter](https://flutter.dev/learn)
 - [Rust](https://www.rust-lang.org/learn)
-- [Appflowy Doumentation](https://appflowy.gitbook.io/docs/essential-documentation/readme)
+- [AppFlowy Doumentation](https://appflowy.gitbook.io/docs/essential-documentation/readme)
 
 <p align="right">(<a href="#introduction">back to top</a>)</p>
 
 <!-- Other Initiatives by Appflowy -->
-## Other Initiatives by Appflowy
+## Other Initiatives by AppFlowy
 
-<p>Here at Appflowy, we foster learning through various initiatives. It's our goal to help anyone who stops by in their developmental journey as much as possible. With this in mind, here are some initiatives by Appflowy for the same: </>
-  
-  - Appflowy Mentorship Program - The AppFlowy mentorship program is aimed at creating a hands-on learning opportunity for new developers who may otherwise lack the opportunity to gain exposure to real-world software development and entry into the technical community. Link: [Appflowy Mentorship Program](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/appflowy-mentorship-program/contributor-guidance)
+Here at AppFlowy, we foster learning through various initiatives. It's our goal to help anyone who stops by in their developmental journey as much as possible. 
+
+With this in mind, here are some initiatives by AppFlowy for the same:
+
+
+  - The [AppFlowy Mentorship Program](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/appflowy-mentorship-program/contributor-guidance) is aimed at creating a hands-on learning opportunity for new developers who may otherwise lack the opportunity to gain exposure to real-world software development and entry into the technical community.
   
   - GitHub Octernships - (Currently Closed)
   
-  - Write for Appflowy - Whether you’re an AppFlowy power user, a software development expert, or just a student starting to get into open source, there is knowledge you can share that will benefit the entire AppFlowy community. Click the link below to learn more. Link: [Write for Appflowy](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/write-for-appflowy)
+  - The [Write for Appflowy](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/write-for-appflowy) initiative allows you to share your knowledge to benefit the entire AppFlowy community, whether you’re an AppFlowy power user, a software development expert, or just a student starting to get into open source.
 
 <p align="right">(<a href="#introduction">back to top</a>)</p>
 
 <!-- Conclusion -->
 ## Conclusion
 
-<p> To summarise, we were able to understand how to contribute to Appflowy, how to handle that first PR, how to connect with the community, and many others in this article. </p>
+To summarize, we were able to understand how to contribute to AppFlowy, how to handle that first PR, how to connect with the community, and many others in this article.
```

Comment:
```suggestion
To summarize, we were able to understand how to contribute to AppFlowy, how to handle that first PR, how to connect with the community, and many others in this article.

Author: [Jonathan Rufus Samuel](https://github.com/JRS296)
Editors: [Eric Phillips](https://github.com/egp415) and [Alex Wallen](https://github.com/a-wallen)
```

Reply:
I will leave it to @annieappflowy how she wants to handle attribution — I see no particular issue with GitHub links at this time, though I believe the idea is that it would be up to the individual author's preference.

I actually feel like the conclusion section needs a bit more work. The phrase "and many others" reads oddly and it lacks a good "call to action" such as linking once again to the list of items to work on.  I would start by rewriting it as follows:

In this article we discussed a variety of subjects such as how to contribute to AppFlowy, how to handle your first PR, and how to connect with the community.

And then something like:

We invite you to start contributing to AppFlowy by selecting your own [issue to work on](https://github.com/AppFlowy-IO/AppFlowy/issues).

Reply:
Yes, I think that sort of nudge would be helpful.

Reply:
> 

Would this work? Let me know if any other changes are needed. Thanks.

Reply:
I would say now that you are now "over writing" — the conclusion should be much more concise.  It does not need to reiterate so much of the article's content.  Here is my suggestion on how to condense it down:
---
You should now be well-equipped to embark on your own open source journey. By following the guidelines in this article, you will be able to successfully resolve pull requests and engage meaningfully with other project contributors.

We hope you participate in AppFlowy's development and make an impact either through coding, bug reporting, feature suggestions, documentation. Contributing to this project and building relationships within the AppFlowy community not only fosters collaboration but also offers opportunities for learning and growth.

We invite you to start contributing to AppFlowy right away by selecting an [[issue to work on](https://github.com/AppFlowy-IO/AppFlowy/issues)](https://github.com/AppFlowy-IO/AppFlowy/issues).

Happy Coding!

Reply:
This works much better. I think we ought to go ahead with your suggestion 👍